apps/system/utils/utils_cpuload.c

```diff
@@ -223,19 +248,21 @@ static void cpuload_print_normal(void)
 	printf("PID | Pri |");
 
 #ifdef CONFIG_SCHED_MULTI_CPULOAD
-	for (int i = PROC_STAT_CPULOAD_SHORT; i <= PROC_STAT_CPULOAD_LONG; i++) {
+	for (int i = PROC_STAT_CPULOAD_SHORT; i <= PROC_STAT_CPULOAD_LONG; i++)
 #endif
+	{
 #ifdef CONFIG_SMP
 		for (int j = 0; j < CONFIG_SMP_NCPUS; j++) {
-			printf("  CPU%d |", j);
+			if (cpu < CONFIG_SMP_NCPUS) {
+				if (cpu == j) {
+					printf("  CPU%d |", j);
+				}
+			} else {
+				printf("  CPU%d |", j);
+			}
```

Comment:
```
if ((cpu >= CONFIG_SMP_NCPUS) || (cpu == j)) {
    printf(....);
}
```
I don't know whether this reduces flash usage or not.

Reply:
Done.